sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -2605,29 +2611,36 @@ pi_result _pi_platform::populateDeviceCacheIfNeeded() {
             Ordinals.push_back(i);
           }
         }
-
-        // Create PI sub-sub-devices with the sub-device for all the ordinals.
-        // Each {ordinal, index} points to a specific CCS which constructs
-        // a sub-sub-device at this point.
-        // FIXME: Level Zero creates multiple PiDevices for a single physical
-        // device when sub-device is partitioned into sub-sub-devices.
-        // Sub-sub-device is technically a command queue and we should not build
-        // program for each command queue. PiDevice is probably not the right
-        // abstraction for a Level Zero command queue.
-        for (uint32_t J = 0; J < Ordinals.size(); ++J) {
-          for (uint32_t K = 0; K < QueueGroupProperties[Ordinals[J]].numQueues;
-               ++K) {
-            std::unique_ptr<_pi_device> PiSubSubDevice(
-                new _pi_device(ZeSubdevices[I], this, PiSubDevice.get()));
-            pi_result Result = PiSubSubDevice->initialize(Ordinals[J], K);
-            if (Result != PI_SUCCESS) {
-              return Result;
+        bool IsPVC =
+            (PiSubDevice->ZeDeviceProperties->deviceId & 0xff0) == 0xbd0;
+
+        // If isn't PVC, then submissions to different CCS can be executed on
+        // the same EUs still, so we cannot treat them as sub-sub-devices.
+        if (IsPVC) {
```

Comment:
shouldn't we continue to expose sub-sub-devices under ExposeCSliceInAffinityPartitioning here?